mopidy_tidal/backend.py

```diff
@@ -27,12 +30,45 @@ def __init__(self, config, audio):
         self.playlists = playlists.TidalPlaylistsProvider(backend=self)
         self.uri_schemes = ['tidal']
 
+    def oauth_login_new_session(self, oauth_file):
+        # create a new session
+        self._session.login_oauth_simple(function=logger.info)
+        if self._session.check_login():
+            # store current OAuth session
+            data = {}
+            data['token_type'] = {'data': self._session.token_type}
+            data['session_id'] = {'data': self._session.session_id}
+            data['access_token'] = {'data': self._session.access_token}
+            data['refresh_token'] = {'data': self._session.refresh_token}
+            with open(oauth_file, 'w') as outfile:
+                json.dump(data, outfile)
+
     def on_start(self):
         quality = self._config['tidal']['quality']
         logger.info("Connecting to TIDAL.. Quality = %s" % quality)
         config = Config(quality=Quality(quality))
         self._session = Session(config)
-        if self._session.login_oauth_simple(function=logger.info):
+        oauth_file = os.path.join(os.getenv("HOME"), '.config/', 'oauth.json')
```

Comment:
Nitpicking maybe, but should this be `tidal-oauth.json` since it's in the shared `.config` folder?

Reply:
Good point, I will update it accordingly. I have yet to update the pypi version, in case you want to test the latest version.

Reply:
@tehkillerbee I pulled the latest and tested it, and it worked. 👍